src/itest/kotlin/io/kotest/provided/ProjectConfig.kt

```diff
@@ -76,9 +80,19 @@ object ProjectConfig : AbstractProjectConfig() {
                         .withStartupTimeout(Duration.ofMinutes(THREE_MINUTES))
                 )
             dockerComposeContainer.start()
-
             logger.info { "Started ZAC Docker Compose containers" }
-
+            logger.info { "Waiting until ZAC is healthy by calling the health endpoint and checking the response" }
+            await.atMost(TEN_SECONDS, TimeUnit.SECONDS)
```

Comment:
You can also use a `Duration` for `atMost`, why not make `TEN_SECONDS` an actual Duration?

Reply:
ah you can.. sorry. I did not see the other `atMost` in IntelliJ initially (methods are not in alphabetical order..)